tasks/landings.js

```diff
@@ -16,6 +17,7 @@ gulp.task('landings', function() {
                     partialsDirectory: ['./src/partials']
                 }))
                 .pipe(rename(path.basename(file.path).replace(/\.handlebars$/, '.html')))
+                .pipe(revReplace({manifest: gulp.src("./tmp/rev/rev-manifest.json")}))
```

Comment:
your revision manifest is calculated only from the css directory. Then, you can have problem if you modify an handlebar with css, or an HTM file, but your with a css directory without any modification.

We have to calculate the manifest revision from all the directory: css + handlebar + html.